modules/article-page/index.js

```diff
@@ -16,25 +16,8 @@ module.exports = {
                 'styles',
                 'bold',
                 'italic',
-                'strike',
```

Comment:
I'm making this different enough to recognize that it's taken precedence from the defaults.